static/app/views/performance/transactionSummary/transactionVitals/index.spec.tsx

```diff
@@ -10,16 +11,27 @@ import {
 } from 'sentry-test/reactTestingLibrary';
 
 import ProjectsStore from 'sentry/stores/projectsStore';
+import {Project} from 'sentry/types';
 import {OrganizationContext} from 'sentry/views/organizationContext';
 import TransactionVitals from 'sentry/views/performance/transactionSummary/transactionVitals';
 import {
   VITAL_GROUPS,
   ZOOM_KEYS,
 } from 'sentry/views/performance/transactionSummary/transactionVitals/constants';
 
-function initialize({project, features, transaction, query} = {}) {
+function initialize({
+  project,
+  features,
+  transaction,
+  query,
+}: {
+  features?: string[];
+  project?: Project;
+  query?: Query;
+  transaction?: string;
+} = {}) {
   features = features || ['performance-view'];
-  project = project || TestStubs.Project();
+  project = project || (TestStubs.Project() as Project);
```

Comment:
interesting that we need to cast here. Perhaps we should fix the type for this fixture.

Reply:
All the fixtures return `any` because of the indirection happening with https://github.com/getsentry/sentry/blob/936fa47370028861b66673f3ba9ffe6f2863a50d/fixtures/js-stubs/types.tsx 

There are way to solve for it, and it'll have to happen as part of this effort. But not in this PR

Reply:
IMO we should leave this without the cast as `any`, since later down the line it will be hard to find this